geode-cq/src/test/java/org/apache/geode/management/internal/cli/commands/ListClientCommandDUnitTest.java

```diff
@@ -1,402 +1,159 @@
-import org.apache.geode.test.junit.categories.FlakyTest;
-import org.apache.geode.test.junit.categories.GfshTest;
-
-@Category({DistributedTest.class, FlakyTest.class, GfshTest.class}) // GEODE-908 GEODE-3530
-@SuppressWarnings("serial")
-public class ListClientCommandDUnitTest extends CliCommandTestBase {
-
-  private final String regionName = "stocks";
-  private int port0 = 0;
-  private int port1 = 0;
-
-  @Test // FlakyTest: GEODE-908
-  public void testListClient() throws Exception {
-    setupSystemForListClient();
-
-    final VM manager = Host.getHost(0).getVM(0);
-
-    String commandString = CliStrings.LIST_CLIENTS;
-    getLogWriter().info("testListClient commandStr=" + commandString);
+import org.apache.geode.test.dunit.rules.ClientVM;
+import org.apache.geode.test.dunit.rules.ClusterStartupRule;
+import org.apache.geode.test.dunit.rules.MemberVM;
+import org.apache.geode.test.junit.rules.GfshCommandRule;
 
-    waitForListClientMbean();
+public class ListClientCommandDUnitTest {
+  @ClassRule
+  public static ClusterStartupRule cluster = new ClusterStartupRule(6);
 
-    final VM server1 = Host.getHost(0).getVM(1);
+  @ClassRule
+  public static GfshCommandRule gfsh = new GfshCommandRule();
 
-    final DistributedMember serverMember = ClientCommandsTestUtils.getMember(server1);
+  private static MemberVM locator, server1, server2;
 
-    String[] clientIds = manager.invoke("get client Ids", () -> {
-      final SystemManagementService service =
-          (SystemManagementService) ManagementService.getManagementService(getCache());
-      final ObjectName cacheServerMBeanName = service.getCacheServerMBeanName(port0, serverMember);
-      CacheServerMXBean bean = service.getMBeanProxy(cacheServerMBeanName, CacheServerMXBean.class);
-      return bean.getClientIds();
-    });
-
-    String serverName = server1.invoke("get distributed member Id",
-        ClientCommandsTestUtils::getDistributedMemberId);
-
-    CommandResult commandResult = executeCommand(commandString);
-    getLogWriter().info("testListClient commandResult=" + commandResult);
-
-    String resultAsString = commandResultToString(commandResult);
-    getLogWriter().info("testListClient resultAsString=" + resultAsString);
-    assertTrue(Result.Status.OK.equals(commandResult.getStatus()));
+  private static ClientVM client1, client2, client3;
 
-    CompositeResultData resultData = (CompositeResultData) commandResult.getResultData();
-    CompositeResultData.SectionResultData section = resultData.retrieveSection("section1");
-    assertNotNull(section);
-    TabularResultData tableResultData = section.retrieveTable("TableForClientList");
-    assertNotNull(tableResultData);
+  @BeforeClass
+  public static void setup() throws Exception {
+    locator = cluster.startLocatorVM(0);
+    int locatorPort = locator.getPort();
+    server1 = cluster.startServerVM(1,
+        r -> r.withRegion(RegionShortcut.REPLICATE, "stocks").withConnectionToLocator(locatorPort));
```

Comment:
"stocks" could be extracted into a final String local variable since it is used throughout these tests.  Makes it easier to change the name of the region down the line if we need to (one place to change)